app/models/manageiq/providers/vmware/infra_manager/provision/state_machine.rb

```diff
@@ -62,28 +62,8 @@ def customize_destination
     signal :autostart_destination
   end
 
-  def autostart_destination
-    if get_option(:vm_auto_start)
-      message = "Starting"
-      _log.info("#{message} #{for_destination}")
-      update_and_notify_parent(:message => message)
-      start_with_cache_reset
-    end
-
-    signal :post_create_destination
-  end
-
   private
 
-  # NOTE: Due to frequent problems with cache not containing the new VM we need to clear the cache and try again.
-  def start_with_cache_reset
-    destination.start
```

Comment:
If this gets removed, what tells the destination to start? (see also [line 62](https://github.com/ManageIQ/manageiq-providers-vmware/pull/506/files#diff-c1cd8b1c816a3091f3105345b15b5e38R62) ☝️ ...what will that call now?)

Reply:
The base class's autostart_destination method, https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_provision/state_machine.rb#L73-L82